packages/bento-design-system/src/SliderField/createSliderField.tsx

```diff
@@ -42,11 +48,26 @@ export function createSliderField({
             onChange: (values: number[]) => props.onChange(values[0]),
           };
 
-    const state = useSliderState({
+    const _state = useSliderState({
       ...props,
       ...valueProps,
       numberFormatter,
     });
+    const state = props.dragStep
+      ? {
+          ..._state,
+          setThumbPercent: (index: number, percent: number) => {
+            const minValue = state.getThumbMinValue(index);
+            const maxValue = state.getThumbMaxValue(index);
+            const value = percent * (maxValue - minValue) + minValue;
+            const roundedValue =
+              roundToStep(roundToStep(value - minValue, props.dragStep!), props.step || 1) +
```

Comment:
```suggestion
              roundToStep(roundToStep(value - minValue, props.dragStep!), props.step ?? 1) +
```

Reply:
(unless you explicitly want to fallback to `1` in case the step is `0`

Reply:
I think we want to fallback to `1`, since it's the internal default if a step is not specified